src/MCPClient/lib/clientScripts/archivematicaClamscan.py

```diff
@@ -259,8 +259,8 @@ def scan_file(file_uuid, path, date, task_uuid):
 
     try:
         size = get_size(file_uuid, path)
-        if not size:
-            logger.error('Unexpected file size')
+        if size is None:
+            logger.error('Getting file size returned: ' + str(size))
```

Comment:
I think it's better to do: `logger.error('Getting file size returned: %s', size)`. What do you think?